app/src/docs/v1.api-spec.yaml

```diff
@@ -1182,6 +1245,52 @@ paths:
           $ref: "#/components/responses/Forbidden"
         default:
           $ref: "#/components/responses/Error"
+  /sync:
+    get:
+      summary: Synchronize with the default bucket
+      description: |-
+        Synchronize the contents of the default bucket with COMS so that it can track and manage objects
+        that are already in the bucket. This avoids the need to re-upload preexisting files through the
+        COMS API.
+
+        It looks at all objects in the bucket and adds them to an internal processing queue;
+        once it is eventually processed, it is tracked by COMS.
```

Comment:
Requests to synchronize the contents of the default S3 bucket directory with COMS. All related objects in the specified bucket will be enumerated and enqueued for processing and tracking by COMS.

Reply:
Editing second paragraph to the following, as discussed earlier:
> This endpoint does not guarantee immediately synchronized results. Synchronization latency will be affected by the remaining number of objects awaiting synchronization.